docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,472 @@
+      "  -0.45621469 -0.35451977 -0.25706215 -0.19350282]\n",
+      " [ 0.03041825  0.19011407  0.31558935  0.45152091  0.57509506  0.57319392\n",
+      "   0.64068441  0.65779468  0.54562738  0.45722433  0.32604563  0.18441065\n",
+      "   0.05038023 -0.03802281 -0.16920152 -0.24239544 -0.27186312 -0.31653992\n",
+      "  -0.31844106 -0.29467681 -0.22718631 -0.19486692 -0.12262357 -0.0769962\n",
+      "  -0.00190114  0.02281369  0.04372624  0.03326996 -0.07414449 -0.05703422\n",
+      "  -0.09315589 -0.18536122 -0.21673004 -0.19011407 -0.18536122 -0.17015209\n",
+      "  -0.10171103 -0.00760456  0.04087452  0.13878327  0.28136882  0.39353612\n",
+      "   0.50665399  0.51140684  0.54847909  0.57889734  0.44676806  0.37927757\n",
+      "   0.32319392  0.06178707 -0.06178707 -0.16159696 -0.28707224 -0.45342205\n",
+      "  -0.55418251 -0.53326996 -0.57984791 -0.49429658 -0.48193916 -0.36501901\n",
+      "  -0.28231939 -0.12642586 -0.02756654  0.09410646  0.17205323  0.23669202\n",
+      "   0.28802281  0.26711027  0.27661597  0.22528517  0.19581749  0.11692015\n",
+      "   0.06463878  0.11501901  0.04277567  0.06749049  0.03992395  0.06273764\n",
+      "   0.13403042  0.16730038  0.18155894  0.24524715  0.23098859  0.25570342\n",
+      "   0.1378327   0.13593156  0.07129278 -0.0960076  -0.16634981 -0.27661597\n",
+      "  -0.38212928 -0.47148289 -0.57984791 -0.53422053 -0.60931559 -0.54277567\n",
+      "  -0.40589354 -0.31368821 -0.2148289   0.0095057 ]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f'The result of the submitted job had {len(result)} PUB and has a value:\\n {result}\\n')\n",
+    "print(f'The associated PubResult of this job has the following DataBins:\\n {result[0].data}\\n')\n",
+    "print(f'And this DataBin has attributes: {result[0].data.keys()}')\n",
+    "print(f'Recall that this shape is due to our array of parameter binding sets having shape (100,), combined with \\n\\\n",
+    "  our array of observables having shape (3, 1), where 2 is the number of parameters in the circuit.\\n')\n",
+    "print(f'The expectation values measured from this PUB are: \\n{result[0].data.evs}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b585649-065c-4e19-a4f4-3f1637143890",
+   "metadata": {},
+   "source": [
+    "### Sampler output\n",
+    "\n",
+    "\n",
+    "The Sampler primitive outputs job results in a similar format, with the exception that each `DataBin` will contain one or more `BitArray` objects that samples of the probability distribution(s), typically one bitstring per shot. The number and attribute label for each bit array object depends on how the `ClassicalRegisters` were defined for the circuit being executed. The measurement data from these `BitArrays` can then be processed into a dictionary with key-value pairs corresponding to each bitstring measured (for example, '1011001') and the number of times (or counts) it was measured.\n",
```

Comment:
"with the exception that each `DataBin` will contain one or more `BitArray` objects that samples of the probability distribution(s)" 

Do the objects sample the probability distributions, or do they contain samples?  I'm confused by "that samples of"

Reply:
They contain the samples for each `ClassicalRegister` defined in the circuit. I changed up the text to clarify this.